h/accounts/__init__.py

```diff
@@ -4,13 +4,12 @@
 
 
 class Error(Exception):
-
-    """Base class for this package's custom exception classes."""
+    """Defines a base class for this package's custom exception classes."""
 
 
 class JSONError(Error):
-
-    """Exception raised when there's a problem with a request's JSON body.
+    """
+    Defines an Exception raised when there's a problem with a request's JSON body.
```

Comment:
IDK, I think this rule (use imperative mood) is good advice but a bad rule to enforce. Anyway when I got feed up with it had done most of them.

Reply:
I thought the imperative was only on methods, not classes? Presumably most classes are a thing, rather than doing a thing?